generic3g/actions/RegridAction.F90

```diff
@@ -165,37 +165,10 @@ function get_regrid_param_from_field_dictionary_(stdname_src, stdname_dst, rc) r
       if (allocated(regrid_method_dst)) then
          tmp_param_dst = EsmfRegridderParam(regridmethod=regrid_method_dst)
       end if
-      param = choose_regrid_param_2_(tmp_param_src, tmp_param_dst, _RC)
+      param = choose_regrid_param_(tmp_param_src, tmp_param_dst, _RC)
```

Comment:
This function needs a bit of thought.    I was thinking that VariableSpec would be the thing that interacts with the FieldDictionary for various reasons.

From the PR it looks like I created this function, but that surprises me give my previous comment.

Reply:
The PR is showing the diff between two of my commits. This is a new function that I introduced.

Reply:
This is handled elsewhere now, in `FieldSpec`. The order in which `regrid_param` of a `FieldSpec` is set is as follows
- Generated from the default regridding method
- Generated from the regridding method specified in `field_dictionary.yml`
- As specified by user during construction